com.github.coslyk.MoonPlayer.json

```diff
@@ -0,0 +1,167 @@
+{
+  "app-id": "com.github.coslyk.MoonPlayer",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "command": "moonplayer",
+  "finish-args": [
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=wayland",
+      "--device=all",
+      "--share=network",
+      "--socket=pulseaudio",
+      "--filesystem=host",
```

Comment:
I think you shouldn't give `host` access to this artifact. Should be `home` or even more preferably (almost all of) the `xdg-*` family.

Reply:
@x80486 Can you stop suggesting `host` -> `home`. It is *not* helpful, all it does is not improve security and annoy users.

Reply:
Then the [Sandbox Permissions](http://docs.flatpak.org/en/latest/sandbox-permissions-reference.html#sandbox-permissions) sections/page is not quite correct and/or intuitive; according to that, `filesystem=home:ro` and `persist=.moonplayer` would be the needed combination for this one. In any case, since you know this better, I wouldn't suggest that again.

Reply:
How is it not correct?

Anyway you can't mix `home` and `persist`, the former overrides the latter.